Dockerfile

```diff
@@ -11,7 +11,9 @@ ENV IS_DOCKER 1
 RUN mkdir -p /pip/{cache,build}
 ADD requirements /pip/requirements
 WORKDIR /pip
-RUN pip install -b /pip/build --download-cache /pip/cache --no-deps -r /pip/requirements/dev.txt --find-links https://pyrepo.addons.mozilla.org/
+# Download this securely from pyprepo first.
+RUN pip install --no-deps --find-links https://pyrepo.addons.mozilla.org/ peep==2.2
```

Comment:
peep 2.4 is available but I don't think you need to pin this to a version. Unlike other libs, this is one you want to keep up to date. I doubt the CLI interface will break but if that happens you could go back and pin it.